dna-frontend/src/app/pages/beneficiaries/beneficiaries.component.scss

```diff
@@ -0,0 +1,41 @@
+:host {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: start;
+  background-color: white;
+  height: 100%;
+}
+
+.nav-tabs {
+  margin-bottom: 1em;
+}
+
+.client-form {
```

Comment:
Should rename this class to beneficiaries-form or put all the form classes in one place since they are shared among multiple pages